scripts/generate_style.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+import * as Style from "../src/js/style.js";
+
+let style = Style.build(
+  "https://6ug7hetxl9.execute-api.us-east-2.amazonaws.com/data/v3.json",
+  "https://zelonewolf.github.io/openstreetmap-americana/sprites/sprite",
+  ["mul"]
```

Comment:
This script could take the language code as a [command line argument](https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/), falling back to `mul` as a neutral default.

We should document the need to install the [mapbox-gl-rtl-text](https://github.com/mapbox/mapbox-gl-rtl-text/) plugin (#590), especially if the language is set to `mul`.

Reply:
Command line arguments for languages with fallback to `mul` implemented in [3c36f42](https://github.com/ZeLonewolf/openstreetmap-americana/pull/674/commits/3c36f42ccb908162180387887e71f793dc78cb6c).

Reply:
Plugin install requirements documented in 9cca82d.

Reply:
Plugin install requirements documented in 9cca82d.